File: 2399.cpp
#include <stdio.h>

using namespace std;

int N;
int x[10000];
long long answer = 0;

int abs(int x) {
    if(x >= 0) return x;
    else return -x;
}

int main(void) {
    scanf("%d", &N);
    for(int i=0; i<N; i++) {
        scanf("%d", &x[i]);
    }
    
    for(int i=0; i<N; i++) {
        for(int j=0; j<N; j++) {
            answer += abs(x[i] - x[j]);
        }
    }
    
    printf("%lld\n", answer);
    return 0;
}

File: 2399.java
package algo;

import java.util.Arrays;
import java.util.Scanner;

class Task {
	private int N;
	private long[] x;
	private final Scanner sc = new Scanner(System.in);
	
	private void input() {
		N = sc.nextInt();
		x = new long[N];
		
		for(int i=0; i<N; i++) {
			x[i] = sc.nextLong();
		}
	}
	
	private void offset() {
		long answer = 0;
		
		for(int i=0; i<N; i++) {
			for(int j=0; j<N; j++) {
				answer += Math.abs((x[i] - x[j]));
			}
		}
		
		System.out.println(answer);
	}
	
	public void run() {
		input();
		offset();
	}
}

public class Main {
	public static void main(String[] args) {
		Task task = new Task();
		task.run();
	}
}
